scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpressionSuggester.scala

```diff
@@ -324,6 +337,16 @@ class SpelExpressionSuggester(
     suggestions
   }
 
+  private def castMethodsSuggestions(klass: Class[_]): Future[Iterable[ExpressionSuggestion]] = {
```

Comment:
let's wrap the whole code into `Future.apply` instead of using `Future.successful` (to make sure that no exception will occur out of the Future context)